scheduler/src/cook/compute_cluster.clj

```diff
@@ -530,11 +530,13 @@
         dissoc-ca-cert-vals
         (fn [configs]
           (map-vals dissoc-ca-cert configs))]
-    (log/info "Updating dynamic clusters."
-              {:current-configs (dissoc-ca-cert-vals current-configs)
-               :force? force?
-               :new-configs (dissoc-ca-cert-vals new-configs)
-               :updates (map #(update % :goal-config dissoc-ca-cert) updates)})
+    (log/info "Updating dynamic clusters.")
```

Comment:
I am in favor of removing this line since the others will provide the same context

Reply:
Makes sense, I kept it mostly for context while making the change, will remove it.